servers/republik/modules/crowdfundings/lib/scheduler/owners.js

```diff
@@ -58,22 +60,27 @@ const getBuckets = async ({ now }, { pgdb }) => {
   // load users with a membership
   const users = await pgdb.query(`
     SELECT
-      DISTINCT(u.*)
+      u.*,
+      json_agg(DISTINCT mt.name) AS "membershipTypes"
     FROM
-      users u
+      memberships m
+    INNER JOIN
+      users u ON m."userId" = u.id
+    INNER JOIN
+      "membershipTypes" mt ON m."membershipTypeId" = mt.id
     WHERE
-      u.id != :PARKING_USER_ID AND
-      u.id IN (
-        SELECT
-          DISTINCT(m."userId")
-        FROM
-          memberships m
-      )
+      m."userId" != :PARKING_USER_ID
+      AND m.active = true
+      AND m.renew = true
+    GROUP BY 1
```

Comment:
why did you change `FROM users` to `FROM memberships`? It's not wrong, I just wonder why you made the change to accomplish the same thing, just a little bit slower.